### ISSUE TITLE
Improve Persian

### ISSUE THREAD

**Issue by musicsd** [florisboard/florisboard#511]
These letters are used in Arabic and other languages; And in Persian have no use and should be removed
`ٱ` `ك` `ي` `ة` `أ` `إ` `◌ً` `◌ٍ` `◌ٌ` `◌ٰ` `◌ٖ` `◌ٓ` 

These letters are used in Persian, and should be added

`ۀ` […](https://en.wiktionary.org/wiki/%DB%80)
`ہ` […](https://en.wiktionary.org/wiki/%DB%81)
`ھ` […](https://en.wiktionary.org/wiki/%DA%BE)
`ے` […](https://en.wiktionary.org/wiki/%DB%92)

**Comment by musicsd** [comment]
@patrickgold

**Comment by patrickgold** [comment]
Thanks for your issue! I haven't added the Persian layout myself and I don't speak Persian, so I can't say whether this letter or that letter is more relevant. Comparing the Gboard Persian layout with the FlorisBoard one though I can say they are at least on the base character level equal, so I am unsure what to do here. The letters you request to be added, should these be popups or base characters?

**Comment by musicsd** [comment]
In the subject of removing ineffectual letter; They are completely inefficient, and they have never been used in Persian, as well as in Persian layer of Samsung's keyboard. Don't really be enough? Show more documents?

**Comment by musicsd** [comment]
In the subject of adding letters:
`ۀ`, `ہ`, and `ھ`, as `ه` popups; And `ے` as `ی` popup.

**Comment by musicsd** [comment]
@patrickgold

**Comment by patrickgold** [comment]
@musicsd Above commit incorporates some of your proposed changes: I added all your proposed characters. For the removal part, I removed one, rearranged some to less relevant positions and for the diacritics I didn't change anything, because I can't copy the code into the Unicode table to check for their code and it is very difficult to distinguish the diacritics else. If you want to further improve the Persian popups, feel free to modify the Persian layout file yourself and make an PR, especially for the diacritics you know exactly which are needed and in which order.
